src/index.ts

```diff
@@ -148,10 +149,13 @@ interface ResolvedColorUsage extends ColorUsage {
   foreground: string
 }
 
-const HEXRegex = /#[0-9a-f]{3,8}\b/gi
-const RGBHSLRegex = /\b(?:rgb|hsl)a?\([\d\s\-,./%]+\)/gi
+const enableDetectColorLangs = ['css', 'scss', 'less', 'html', 'vue', 'svelte', 'md']
```

Comment:
I think for now it would be safer to limited to CSS-like languages first. Support `md` `html` would be too risky to have false positive.

Reply:
I'll replace it with: ['css', 'less', 'scss', 'sass', 'stylus']

This includes almost all the mainstream preprocessor languages on the market at present.